app/src/main/java/com/eventyay/organizer/core/event/list/EventListFragment.java

```diff
@@ -59,6 +64,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         // Inflate the layout for this fragment
         binding = DataBindingUtil.inflate(inflater, R.layout.event_list_fragment, container, false);
 
+        appBarLayout = getActivity().findViewById(R.id.app_bar_main);
```

Comment:
You already have the binding, no need of this

Reply:
@iamareebjamal data binding would let me access the layout elements that are present in the inflated layout and not the app bar layout. Hence this needs to be done.